xinference/model/llm/llm_family.json

```diff
@@ -5588,5 +5588,94 @@
         32000
       ]
     }
-  }
+  },
+  {
+    "version": 1,
+    "context_length": 32768,
+    "model_name": "internvl−chat−v1-5",
```

Comment:
How about calling it `internvl-1.5-chat`?

Reply:
<img width="874" alt="image" src="https://github.com/xorbitsai/inference/assets/128140880/fa7fee02-6d62-483a-baf2-2d71bd464245">
1.5 is considered a separator when importing a model, what about internvl-1-5-chat？